oauth2client/transport/httplib2.py

```diff
@@ -29,6 +29,9 @@
 _MAX_REFRESH_ATTEMPTS = 2
 
 
+exceptions = (httplib2.HttpLib2Error,)
```

Comment:
I immediately regret requesting it go in the transport. Can you instead just define `exceptions` as `None` on `TransportComplianceTests` and make the subclasses update the value?

This is a burden that shouldn't be public.

(I promise this is my last request.)